src/app/MessageDef/InvokeCommand.cpp

```diff
@@ -61,7 +61,6 @@ CHIP_ERROR InvokeCommand::Parser::CheckSchemaValidity() const
     CommandList::Parser commandList;
 
     PRETTY_PRINT("InvokeCommand =");
-    PRETTY_PRINT("{");
```

Comment:
nit: is this intentional? still "}" is printed after the `while`.

Reply:
this is accidental remove, still needed, thanks.